router/router.go

```diff
@@ -0,0 +1,29 @@
+package router
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/rayl17/academy-go-q42021/controller"
+	parameters "github.com/rayl17/academy-go-q42021/global"
+)
+
+func NewRouter(c controller.ControllerInterface) *http.Server {
+
+	router := mux.NewRouter()
+
+	router.HandleFunc("/pokemon/{name}", c.GetPokemonByName).Methods("GET")
+	router.HandleFunc("/pokemonid/{id}", c.GetPokemonByID).Methods("GET")
+	router.HandleFunc("/pokemon/{name}", c.PostPokemon).Methods("POST")
+	router.HandleFunc("/concurrency/", c.GetPokemonsConcurrently).Methods("GET")
```

Comment:
concurrency whay? A route should be self-descriptive to denote what entity and action are performed.

```suggestion
	router.HandleFunc("/pokemon/concurrency/", c.GetPokemonsConcurrently).Methods("GET")
```